src/battle_script_commands.c

```diff
@@ -1925,7 +1941,7 @@ s32 CalcCritChanceStageArgs(u32 battlerAtk, u32 battlerDef, u32 move, bool32 rec
                     + gMovesInfo[move].criticalHitStage
                     + (holdEffectAtk == HOLD_EFFECT_SCOPE_LENS)
                     + 2 * (holdEffectAtk == HOLD_EFFECT_LUCKY_PUNCH && gBattleMons[battlerAtk].species == SPECIES_CHANSEY)
```

Comment:
I don't know that it's worth having `IsBattlerAffectedByLuckyPunch` for just Chansey given that it doesn't apply to Blissey, but if we ever wanted to separate that out stylistically for future proofing this seems like a good opportunity. Up to you idk if it's worth it.